services/API-service/src/api/event/event.service.ts

```diff
@@ -583,8 +583,8 @@ export class EventService {
       .select([
         'triggerPerLeadTime.leadTime as "leadTime"',
         'triggerPerLeadTime.date as date',
-        'MAX(CASE WHEN triggerPerLeadTime.triggered = TRUE THEN 1 ELSE 0 END) as triggered',
-        'MAX(CASE WHEN triggerPerLeadTime.thresholdReached = TRUE THEN 1 ELSE 0 END) as "thresholdReached"',
+        'MAX(CASE WHEN triggerPerLeadTime.forecastAlert = TRUE THEN 1 ELSE 0 END) as triggered',
+        'MAX(CASE WHEN triggerPerLeadTime.forecastTrigger = TRUE THEN 1 ELSE 0 END) as "thresholdReached"',
```

Comment:
In case you missed this, the alias still uses the old name - `as thresholdReached`

Reply:
same